<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-7GVQ1KM" ignored="0" name="Box Test Cases" tests="7" failures="3" timestamp="19 May 2021 16:02:08 GMT" time="146.334" errors="0">
  <testcase name="addNewFolder" time="25.41" classname="com.payBox.testCases.TC_HomeTest"/>
  <testcase name="uploadFile" time="32.659" classname="com.payBox.testCases.TC_UploadTest"/>
  <testcase name="loginTest" time="29.021" classname="com.payBox.testCases.TC_LoginTest_DDT"/>
  <testcase name="loginTest" time="8.446" classname="com.payBox.testCases.TC_LoginTest_DDT">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.payBox.testCases.TC_LoginTest_DDT.loginTest(TC_LoginTest_DDT.java:32)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="5.873" classname="com.payBox.testCases.TC_LoginTest_DDT">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.payBox.testCases.TC_LoginTest_DDT.loginTest(TC_LoginTest_DDT.java:32)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="8.408" classname="com.payBox.testCases.TC_LoginTest_DDT">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.payBox.testCases.TC_LoginTest_DDT.loginTest(TC_LoginTest_DDT.java:32)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="15.679" classname="com.payBox.testCases.TC_LoginTest_DDT"/>
</testsuite> <!-- Box Test Cases -->
